### DISCUSSION TITLE
Help me understand how the app is behaving?

### DISCUSSION THREAD

**Discussion by Sea-Worker9552** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#605] [Q&A] [answered]
Hello! New AlDente Pro user here. I just started trying it out and I have a few questions. For reference I'm using this on a new 14" M1 Pro MBP running the latest version of OS X Monterey (12.2.1)

1. The app tells me I should disable Apple's charging optimizations. I did not do that, but set my AlDente limit at 70%, which I see it obeying. My Mac is not charging pas 70%, which is what I want. However whether I toggle or de-toggle the charging optimization option, I don't see any behavior changes. **What are the consequences if I don't disable Apple's charging optimizations?**
2. I don't have "Stop charging when sleeping" or "Stop charging when app closed" checked, but I noticed that when I put my MacBook to sleep (Apple Menu > Sleep), my Mac holds 70% just fine. I don't see the device charge to 100%. My reading of the help documentation seems to suggest that in sleep my Mac will start charging. Do I need to do a disconnect/reconnect charging event to trigger charging to start or something? I guess my point is the documentation is really clear on what I need to do, but somehow my Mac is already behaving in a way that I would want to only if I toggle those settings, but without actually toggling those settings. This is confusing to me.

**Comment by MatthiasKerbl** [comment, marked as answer]
Hi @Sea-Worker9552,

Ad 1.: Some users experienced charging past the charge limit when Apples optimized charging and sometimes it results in a loop of charging on/charging off. That’s why we recommend turning it off.

Ad2.: When your MacBooks charging is already paused when it goes into sleep then charging will stay paused. When it is still charging to the charge limit and it goes into sleep, it will charge past the charge limit to 100%.

Cheers,
Matthias

**Comment by Sea-Worker9552** [comment]
Thanks. This is super clear. I appreciate the explanation.

**Comment by Bachsau** [comment]
You have been told to disable it, so why don't you do it? If you have two programs messing with the same controls, they will get in each other's way. Isn't that obvious?
